examples/sph/run_sod_dust_tva.py

```
@@ -0,0 +1,299 @@
+
+        model.timestep()
+
+
+Simulation(model).run()
+
+# %%
+
+glob_str = f"{dump_folder}sod_*.png"
+ani = show_image_sequence(glob_str)
+
+writer = PillowWriter(fps=15, metadata=dict(artist="Me"), bitrate=1800)
```

Ruff (0.15.21)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)